notebooks/MAST/TESS/beginner_tess_tap_search/requirements.txt

```diff
@@ -0,0 +1 @@
+astropy>=0.3.9
```

Comment:
Oops, missed this out of my review. Add requests, astroquery, and ipython to this file.

Reply:
Added current versions from my environment. They may be newer than necessary.